packages/router/src/Set.tsx

```diff
@@ -45,10 +47,16 @@ export function Set<WrapperProps>(props: SetProps<WrapperProps>) {
     unauthenticated,
     roles,
     whileLoadingAuth,
+    whileLoadingPage,
     ...rest
   } = props
   const routerState = useRouterState()
-  const { loading, isAuthenticated, hasRole } = routerState.useAuth()
+  const {
+    loading: authLoading,
+    isAuthenticated,
+    hasRole,
+  } = routerState.useAuth()
+  const { loading: pageLoading } = usePageLoadingContext()
```

Comment:
This line causes tests to fail... still unsure why.

Reply:
@Tobbe Still a bit stuck on this, would love to know if you have any feedback, or have seen [errors like these](https://cloud.nx.app/runs/SFh6eO3T8J/task/@redwoodjs%2Frouter:test) before.